.cirrus.yml

```
@@ -62,3 +62,83 @@ task:
   make_test-tool_script: make test-tool
   make_test-all_script: make test-all
   make_test-spec_script: make test-spec
+
+
+# The following is specific to Shopify/ruby and should be submitted upstream
+yjit_task:
+  name: Arm64 Graviton2 / $CC YJIT New Backend Temp Checks
```

```suggestion
  name: Arm64 Graviton2 / $CC YJIT Checks
```

As you were saying we should probably stop saying it's temp checks.